<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AntCtrl</class>
 <widget class="QWidget" name="AntCtrl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1576</width>
    <height>857</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>466</width>
    <height>381</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Тест антенны</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_8">
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="formLayoutWidget">
      <layout class="QGridLayout" name="gridLayout_7">
       <item row="1" column="0">
        <spacer name="vSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>498</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="maximumSize">
          <size>
           <width>460</width>
           <height>450</height>
          </size>
         </property>
         <property name="title">
          <string>Управление антенной</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="0" colspan="5">
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="3">
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Minimum</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>238</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="0" column="1">
             <widget class="QPushButton" name="powerButton">
              <property name="toolTip">
               <string>Питание</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QComboBox" name="port">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>32</height>
               </size>
              </property>
              <property name="toolTip">
               <string>Порт подключения антенны</string>
              </property>
              <item>
               <property name="text">
                <string>/dev/ttyS0</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>/dev/ttyS1</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>/dev/ttyS2</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>/dev/ttyS3</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>/dev/ttyUSB0</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>/dev/ttyUSB1</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>/dev/ttyUSB2</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>/dev/ttyUSB3</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QLabel" name="powerLabel">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="0">
           <widget class="QToolButton" name="showCorr">
            <property name="text">
             <string>...</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="4">
           <widget class="QPushButton" name="showTableButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="layoutDirection">
             <enum>Qt::RightToLeft</enum>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="5">
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="0" column="0">
             <layout class="QGridLayout" name="gridLayout_3">
              <item row="0" column="0">
               <layout class="QGridLayout" name="gridLayout">
                <item row="5" column="0" rowspan="2">
                 <widget class="QLabel" name="label_4">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>32</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Прогон, °/с:</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                  </property>
                 </widget>
                </item>
                <item row="6" column="1">
                 <spacer name="horizontalSpacer_4">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeType">
                   <enum>QSizePolicy::Minimum</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>28</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item row="6" column="2">
                 <widget class="QDoubleSpinBox" name="azStep">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>28</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>61</width>
                    <height>32</height>
                   </size>
                  </property>
                  <property name="toolTip">
                   <string>Скорость прогона по азимуту</string>
                  </property>
                  <property name="decimals">
                   <number>1</number>
                  </property>
                  <property name="minimum">
                   <double>1.000000000000000</double>
                  </property>
                  <property name="maximum">
                   <double>9.000000000000000</double>
                  </property>
                  <property name="singleStep">
                   <double>1.000000000000000</double>
                  </property>
                  <property name="value">
                   <double>3.000000000000000</double>
                  </property>
                 </widget>
                </item>
                <item row="6" column="3">
                 <widget class="QDoubleSpinBox" name="elStep">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>28</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>61</width>
                    <height>32</height>
                   </size>
                  </property>
                  <property name="toolTip">
                   <string>Скорость прогона по углу места</string>
                  </property>
                  <property name="decimals">
                   <number>1</number>
                  </property>
                  <property name="minimum">
                   <double>1.000000000000000</double>
                  </property>
                  <property name="maximum">
                   <double>10.000000000000000</double>
                  </property>
                  <property name="singleStep">
                   <double>1.000000000000000</double>
                  </property>
                  <property name="value">
                   <double>3.000000000000000</double>
                  </property>
                 </widget>
                </item>
                <item row="6" column="4">
                 <widget class="QPushButton" name="runBtn">
                  <property name="minimumSize">
                   <size>
                    <width>61</width>
                    <height>32</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>61</width>
                    <height>32</height>
                   </size>
                  </property>
                  <property name="toolTip">
                   <string>Запуск прогона</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="shortcut">
                   <string/>
                  </property>
                  <property name="checkable">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item row="2" column="4">
                 <widget class="QPushButton" name="currBtn">
                  <property name="minimumSize">
                   <size>
                    <width>61</width>
                    <height>32</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>61</width>
                    <height>32</height>
                   </size>
                  </property>
                  <property name="toolTip">
                   <string>Обновить текущее положение</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="shortcut">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item row="4" column="4">
                 <widget class="QPushButton" name="setButton">
                  <property name="text">
                   <string/>
                  </property>
                  <property name="checkable">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item row="0" column="2">
                 <widget class="QLabel" name="TL_azimutCorrect_4">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>28</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>57</width>
                    <height>28</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Азимут</string>
                  </property>
                  <property name="wordWrap">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
                <item row="0" column="3">
                 <widget class="QLabel" name="TL_elevatCorrect_4">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>28</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Угол места</string>
                  </property>
                  <property name="wordWrap">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0" rowspan="2" colspan="2">
                 <widget class="QLabel" name="label_2">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>32</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Текущее положение, °:</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                  </property>
                 </widget>
                </item>
                <item row="1" column="2" rowspan="3">
                 <widget class="QLineEdit" name="azCur">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>28</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>61</width>
                    <height>32</height>
                   </size>
                  </property>
                  <property name="toolTip">
                   <string>Текущий азимут</string>
                  </property>
                  <property name="readOnly">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item row="1" column="3" rowspan="3">
                 <widget class="QLineEdit" name="elCur">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>28</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>61</width>
                    <height>32</height>
                   </size>
                  </property>
                  <property name="toolTip">
                   <string>Текущий угол места</string>
                  </property>
                  <property name="readOnly">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1" rowspan="2">
                 <spacer name="horizontalSpacer">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeType">
                   <enum>QSizePolicy::Minimum</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>28</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item row="3" column="0" rowspan="2">
                 <widget class="QLabel" name="label_5">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>32</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Целевое положение°:</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                  </property>
                 </widget>
                </item>
                <item row="4" column="1">
                 <spacer name="horizontalSpacer_5">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeType">
                   <enum>QSizePolicy::Minimum</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>28</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item row="4" column="2">
                 <widget class="QDoubleSpinBox" name="azDest">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>28</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>61</width>
                    <height>32</height>
                   </size>
                  </property>
                  <property name="toolTip">
                   <string>Наведение азимута</string>
                  </property>
                  <property name="decimals">
                   <number>1</number>
                  </property>
                  <property name="minimum">
                   <double>0.000000000000000</double>
                  </property>
                  <property name="maximum">
                   <double>360.000000000000000</double>
                  </property>
                 </widget>
                </item>
                <item row="4" column="3">
                 <widget class="QDoubleSpinBox" name="elDest">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>28</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>61</width>
                    <height>32</height>
                   </size>
                  </property>
                  <property name="toolTip">
                   <string>Наведение угла места</string>
                  </property>
                  <property name="decimals">
                   <number>1</number>
                  </property>
                  <property name="minimum">
                   <double>0.500000000000000</double>
                  </property>
                  <property name="maximum">
                   <double>179.000000000000000</double>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="1" column="0">
               <widget class="QPushButton" name="followButton">
                <property name="toolTip">
                 <string>Сопровождать спутник по расписанию</string>
                </property>
                <property name="text">
                 <string>Сопровождение по расписанию</string>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="1" column="0">
             <widget class="QGroupBox" name="sessionGbox">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="title">
               <string>Ближайшая сессия</string>
              </property>
              <layout class="QVBoxLayout" name="iSessionVbox">
               <item>
                <layout class="QHBoxLayout" name="iRecvSatHbox">
                 <property name="leftMargin">
                  <number>6</number>
                 </property>
                 <property name="rightMargin">
                  <number>6</number>
                 </property>
                 <item>
                  <widget class="QLabel" name="iRecvSatTitleLbl">
                   <property name="text">
                    <string>Спутник:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="recvSatLbl">
                   <property name="text">
                    <string>FENGYUN 3A</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="iSessionTimerHbox">
                 <property name="leftMargin">
                  <number>6</number>
                 </property>
                 <property name="rightMargin">
                  <number>6</number>
                 </property>
                 <item>
                  <widget class="QLabel" name="sessionTimerTitleLbl">
                   <property name="text">
                    <string>До начала:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="sessionTimerLbl">
                   <property name="text">
                    <string>1:08:40</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="3">
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>204</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="2">
           <widget class="QPushButton" name="setCorrButton">
            <property name="toolTip">
             <string>Установить угол коррекции</string>
            </property>
            <property name="text">
             <string>Установить</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QDoubleSpinBox" name="corrSpinBox">
            <property name="toolTip">
             <string>Угол коррекции</string>
            </property>
            <property name="minimum">
             <double>-99.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QGroupBox" name="tableBox">
         <property name="title">
          <string>Расписание пролёта спутников</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="tableLayout"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
